### PR TITLE
country restriction and ssv-keys verion error

### PR DESCRIPTION
**Pull request by axelrod-blox**
included Israel into country restrictions and added error when for unsupported versions

### CODE REVIEW

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts [hunk 1]
```diff
@@ -210,9 +210,14 @@ class OperatorStore extends BaseStore {
...
   async updateOperatorValidatorsLimit(): Promise<void> {
     const contract = getContractByName(EContractName.GETTER);
-    if (this.operatorValidatorsLimit === 0) {
-      this.operatorValidatorsLimit = await contract.getValidatorsPerOperatorLimit();
+    try {
+      if (this.operatorValidatorsLimit === 0) {
+        this.operatorValidatorsLimit = await contract.getValidatorsPerOperatorLimit();
+      }
+    } catch (e) {
+      console.log(e);
```
[the remark this answers is not in the document]
Comment (alexstolr) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts:218]: I see it mostly effects the next route redirect via setting applicationStore.strategyRedirect, and defines the pagination behavior when setting the url. You can catch at the caller of this (updateOperatorValidatorsLimit) function and said this values based on what we wnat (as well as console log). Move it there pls

**Review by alexstolr — CHANGES_REQUESTED** [review]
I still didn't finish the review but you can start handling my first comment while I review the rest.

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexstolr) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts:218]: btw, which route we should take in scenario is a product decision, so best consult with Ariel.

[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts]: the catch case can happening if we provide incorrect contract address, changed this part with new message, 
I think we should speak with Ariel to create error page and show him in cases like this

[the remark this answers is not in the document]
Comment (alexstolr) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts]: Actually I don't remember how I reached my above conclusion.

**Review by alexstolr — APPROVED** [review]

**Pull request merged**